ludwig/utils/hyperopt_utils.py

```diff
@@ -23,6 +24,7 @@
 from abc import ABC, abstractmethod
 from typing import Any, Dict, List
 
+import fiber
```

Comment:
Can yu move this import in the init of the FiberExecutor? This way if one does not need to use the fiber executor there will be no import error. I believe you refer to fiber directly only in the init to create the pool

Reply:
sure, will do.